pkg/state/state_hostdevice_network.go

```diff
@@ -38,6 +38,8 @@ import (
 )
 
 const (
+	stateHostDeviceCNIVersion         = "v0.3.1"
```

Comment:
@rollandf I think this is why the e2e tests fail. I should be specifying cniVersion as `0.3.1` instead of `v0.3.1`